controllers/submariner/submariner_metrics.go

```diff
@@ -38,6 +40,15 @@ var (
 			Help: "Number of gateways",
 		},
 	)
+	gatewayUptimeGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "submariner_gateway_uptime",
+			Help: "Timestamp of gateway creation time",
```

Comment:
Shouldn't up time be the time elapsed since creation?

Reply:
I'll change the name to reflect the metric data.
It is recommended by Prometheus to capture timestamp vs. time since